docs/2fa/users/guide.md

```diff
@@ -0,0 +1,100 @@
+# User Guide
+
+<!-- Visit [Automata 2FA Guru](https://app.2fa.guru) to start your 2FA protection. -->
+Visit Automata 2FA Guru to start your 2FA protection.
+
+---
+
+## Setup 2FA
+<!-- - Visit [Automata 2FA Guru](https://app.2fa.guru). -->
+- Visit Automata 2FA Guru.
+
+- Use metamask and connect your wallet.
+
+- If your account hasn't signed up, you can click the **Wallet Actions** button on the top, and go through the signup process. You need to sign a message to prove your identity before generating a 2FA secret. Attention, you must use a different device to store the 2FA secret, don't keep the 2FA device and your wallet in the same one.
+
+![](../../assets/2fa/setup-1.png)
+
+![](../../assets/2fa/setup-2.png)
+
+![](../../assets/2fa/setup-3.png)
+
+And we also support 1password for the 2FA secret representation. You need to prepare a **Login** item in 1password, configure the basic information, then you can use **Scan QR Code** function in 1password. It will make it more flexiable and efficient.
+
+![](../../assets/2fa/1password.png)
+
+Recommend configuration in 1password
+
+| | |
+| --- | --- |
+| username | wallet address |
+| website | https://app.2fa.guru |
+| tags | the target network |
+
+- Once you successfully sign up, we recommend you add a recovery wallet for your primary account, which can be used to reset your 2FA device. Otherwise, if you lost your 2FA device, then the only thing you can do is to disable the 2FA protection before you use your 2FA protected assets.
```

Comment:
Once you've successfully signed up, we recommend that you add a recovery wallet for your primary wallet, which can be used to reset your 2FA device. Without a recovery wallet, you can only disable the 2FA protection before you use your 2FA protected assets.

Reply:
I will refine it to `Without a recovery wallet, if you lost your 2FA device, then you can only disable the 2FA protection before you use your 2FA protected assets.` for the last sentence.